features/barprop.yml

```diff
@@ -0,0 +1,7 @@
+name: BarProp
+description: "BarProp is implemented by browser UI such as the address bar or scrollbars, historically to represent whether the user interface component was visible. For privacy reasons, the spec was changed to make `visible` represent whether is a popup or not. Apple has not yet implemented the change."
+spec: https://html.spec.whatwg.org/multipage/nav-history-apis.html#barprop
+compat_features:
+  - api.BarProp
+  - api.BarProp.visible
+  - api.BarProp.visible.returns_popup
```

Comment:
This should include the APIs that actually return this interface (because you can construct a `BarProp` and you wouldn't even if you could). This is going to be `api.Window.locationbar`, etc. as seen in non-normative note in the spec: https://html.spec.whatwg.org/multipage/nav-history-apis.html#browser-interface-elements